src/components/LayoutHeader/LanguageDropdown.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {Link} from 'gatsby';
+
+import {Menu, MenuButton, MenuList, MenuLink} from '@reach/menu-button';
+import '@reach/menu-button/styles.css';
+
+import {colors} from 'theme';
+import languages from '../../../content/languages.yml';
+
+const completeLanguages = languages.filter(language => {
+  return language.status == 2;
+});
+
+function getPageLink(code, pathname) {
+  const prefix = code === 'en' ? '' : `${code}.`;
+  return `https://${prefix}reactjs.org${pathname}`;
+}
+
+export default function LanguageDropdown({children, location}) {
+  const linkStyle = {
+    ':hover': {
+      color: `${colors.brand} !important`,
+      backgroundColor: 'transparent !important',
+    },
+    ':focus': {
+      backgroundColor: `${colors.lighter} !important`,
+    },
+  };
+  return (
+    <Menu>
+      <MenuButton
+        css={{
+          background: 'none',
+          color: 'white',
+          border: 'none',
+          cursor: 'pointer',
+        }}>
+        {children}
+      </MenuButton>
+      <MenuList
+        css={{
+          backgroundColor: `${colors.darker} !important`,
+          color: 'white !important',
+          fontSize: '1rem !important',
+          border: 'none !important',
+        }}>
+        {completeLanguages.map(lang => (
+          <MenuLink
+            css={linkStyle}
+            key={lang.code}
+            target="_blank"
```

Comment:
Can remove it? Like this at `developer.mozilla.org`...

Reply:
The difference is that in MDN, all the languages are under the same domain. vuejs.org opens a new window for each language, since it's technically different subdomains.